mock/py/mockbuild/backend.py

```diff
@@ -431,6 +434,13 @@ def copy_srpm_into_chroot(self, srpm_path):
         shutil.copyfile(srpm_path, os.path.join(dest, srpmFilename))
         return os.path.join(self.buildroot.builddir, 'originals', srpmFilename)
 
+    @traceLog()
+    def copy_spec_into_chroot(self, spec_path):
+        specFilename = os.path.basename(spec_path)
+        dest = self.buildroot.make_chroot_path(self.buildroot.builddir, 'originals')
+        shutil.copyfile(spec_path, os.path.join(dest, specFilename))
```

Comment:
I would say that shutil.copy2() would be a better choice. As the target file will not be owned by root:root but likely by the non-privileged user.